### ISSUE TITLE
Add compile / install instructions

### ISSUE THREAD

**Issue by AberDerBart** [blinry/nutsh#2]
There are no instructions on how to compile and / or install `nutsh`. Especially for beginners on the command line this makes things more complicated than necessary.

**Comment by blinry** [comment]
Agreed! Here's the installation instructions I wrote back then: http://nutsh.morr.cc

I'm pretty surprised this still seems to work! :O Some chapters are missing, though.
